charts/vertical-pod-autoscaler/CHANGELOG.md

```diff
@@ -20,6 +20,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED]
 
+## [v1.5.0] - 2024-01-05
+
```

Comment:
```suggestion
```

The version will be set when this is released.

Reply:
Could you please remove this line?

Reply:
All set.